src/Components/Popover.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { cloneElement } from "react"
+import styled from "styled-components"
+
+import * as fonts from "../Assets/Fonts"
+
+interface Props extends React.HTMLProps<HTMLDivElement> {
+  placement?: "top" | "right" | "bottom" | "left"
+  arrowOffsetTop?: string
+  arrowOffsetLeft?: string
+}
+
+export class Popover extends React.Component<Props, null> {
+  private readonly Arrows = {
+    top: <PopoverTopArrow />,
+    right: <PopoverRightArrow />,
+    bottom: <PopoverBottomArrow />,
+    left: <PopoverLeftArrow />,
+  }
+
+  render() {
+    const arrowElement = this.Arrows[this.props.placement]
+    const { arrowOffsetLeft, arrowOffsetTop } = this.props
+
+    return (
+      <PopoverContainer {...this.props as any}>
+        {cloneElement(arrowElement, { arrowOffsetLeft, arrowOffsetTop })}
+
+        <PopoverInner>{this.props.children}</PopoverInner>
+      </PopoverContainer>
+    )
+  }
+}
+
+const PopoverContainer = styled.div`
+  text-align: left;
+  text-align: start;
+  position: relative;
+  padding: 5px;
+  display: inline-block;
+`
+
+const PopoverArrow: any = styled.div`
```

Comment:
using `any` just to make the CI happy. Open to suggestions as to what type this should be.